src/Container.php

```diff
@@ -744,6 +760,14 @@ static function (self $container) use (
                     $threadCount,
                     $dryRun
                 );
+
+                $junitLogPath = $configuration->getJunitLogPath();
+
+                if ($junitLogPath !== null) {
+                    $container->defaultJUnitPath = $junitLogPath;
```

Comment:
Is this used anywhere?

Reply:
[Here](https://github.com/vkhramtsov/infection/blob/feature-adjust-handling-of-predefined-xml-coverage-and-junit-log/src/Container.php#L793)?

Reply:
What do you think about doing this in `getDefaultJUnitFilePath`?

Reply:
From my point of view it will be overhead to get configuration from container in `getDefaultJUnitFilePath` method, but if you insisting...

Reply:
Is it not memoized there is it?

Reply:
Let's wait for review from @maks-rafalko. After that I will change this in case it will be still actual.

Reply:
Here I agree with @sanmai - it's better to move it to `getDefaultJUnitFilePath()`. The logic will be in one place (method) and we won't have state mutation outside of `getDefaultJUnitFilePath()` - it will be just memoization saved into the object state.

Reply:
Changed